easy-ui-react/src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,83 @@
+@use "../styles/common" as *;
+
+// prettier-ignore
+.statusPromotional {
+  @include component-token("notification", "color", theme-token("color.text.inverse"));
+  @include component-token("notification", "color.background", theme-token("color.background.primary"));
+}
+
+// prettier-ignore
+.statusSuccess {
+  @include component-token("notification", "color", theme-token("color.text.success.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.success.subtle"));
+}
+
+// prettier-ignore
+.statusNeutral {
+  @include component-token("notification", "color", theme-token("color.text.secondary.bold"));
+  @include component-token("notification", "color.background",theme-token("color.background.secondary.subtle"));
+}
+
+// prettier-ignore
+.statusWarning {
+  @include component-token("notification", "color", theme-token("color.text.inverse.inverse.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.warning.subtle"));
+}
+
+// prettier-ignore
+.statusError {
+  @include component-token("notification", "color", theme-token("color.text.danger.bold"));
+  @include component-token("notification", "color.background", theme-token("color.background.danger.subtle"));
+}
+
+.typeToast {
+  display: inline-flex;
+}
+
+.typeAlert {
+  display: flex;
+  width: 100%;
+}
+
+// prettier-ignore
+.Notification {
+  @include component-token("notification", "border_radius", design-token("shape.border_radius.base"));
+  @include component-token("notification", "max-width", 1200px);
+  align-items: center;
+  justify-content: space-between;
+  margin: auto;
+  padding: design-token("space.1");
+  border-radius: 0 0 component-token("notification", "border_radius") component-token("notification", "border_radius");
+  max-width: component-token("notification", "max-width");
+  color: component-token("notification", "color");
+  background-color: component-token("notification", "color.background");
+}
+
+.iconText {
+  display: inline-flex;
+  align-items: center;
+  gap: design-token("space.1");
+}
+
+// prettier-ignore
```

Comment:
**nit:** would it make sense for some of these cases to be a little more selective with the `// prettier-ignore`?

Reply:
if i understand correctly, i added `/* prettier-ignore */` instead of `// prettier-ignore`, which i think has block scope, but let me know if i misunderstood the suggestion

Reply:
i think the way prettier ignore works is that it ignores the block that it's attached to. so if it's above a selector it'll apply to the whole block and if it's just above a line it'll apply to that line. so i was just meaning we could selectively apply it to lines instead of whole blacks to limit the scope of the ignore. but seems like it's pretty good now